scripts/irods/test/test_iadmin.py

```diff
@@ -730,6 +730,48 @@ def test_rebalance_for_repl_node(self):
         self.admin.assert_icommand("iadmin rmresc repl")
         self.admin.assert_icommand("iadmin rmresc pt")
 
+    def test_rebalance_for_tree_with_many_resources__issue_5110(self):
+        hostname = lib.get_hostname()
+
+        try:
+            # =-=-=-=-=-=-=-
+            # STANDUP
+            self.admin.assert_icommand("iadmin mkresc repl replication", 'STDOUT_SINGLELINE', "Creating")
```

Comment:
Feel free to ignore this comment, but I wanted to bring it to your attention.

Consider using the convenience functions shown below for creating and manipulating resource hierarchies. You may like them.
https://github.com/irods/irods/blob/4551bf88c0bd481691ffcf56a823ca58fad3d317/scripts/irods/lib.py#L686
https://github.com/irods/irods/blob/4551bf88c0bd481691ffcf56a823ca58fad3d317/scripts/irods/lib.py#L696
https://github.com/irods/irods/blob/4551bf88c0bd481691ffcf56a823ca58fad3d317/scripts/irods/lib.py#L699
https://github.com/irods/irods/blob/4551bf88c0bd481691ffcf56a823ca58fad3d317/scripts/irods/lib.py#L702
https://github.com/irods/irods/blob/4551bf88c0bd481691ffcf56a823ca58fad3d317/scripts/irods/lib.py#L705
https://github.com/irods/irods/blob/4551bf88c0bd481691ffcf56a823ca58fad3d317/scripts/irods/lib.py#L708

Reply:
will do.

Reply:
done